src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -622,229 +622,142 @@ static GetItemEntry sGetItemTable[] = {
-    &gPlayerAnim_002DE8, &gPlayerAnim_002DE8, &gPlayerAnim_002DE8,
-};
-
-static LinkAnimationHeader* D_80853C14[] = {
-    &gPlayerAnim_002E30, &gPlayerAnim_002E18, &gPlayerAnim_002E18,
-    &gPlayerAnim_002E30, &gPlayerAnim_002E30, &gPlayerAnim_002E30,
-};
-
-static LinkAnimationHeader* D_80853C2C[] = {
-    &gPlayerAnim_002E40, &gPlayerAnim_002E38, &gPlayerAnim_002E38,
-    &gPlayerAnim_002E40, &gPlayerAnim_002E40, &gPlayerAnim_002E40,
-};
-
-static LinkAnimationHeader* D_80853C44[] = {
-    &gPlayerAnim_002E28, &gPlayerAnim_002E20, &gPlayerAnim_002E20,
-    &gPlayerAnim_002E28, &gPlayerAnim_002E28, &gPlayerAnim_002E28,
-};
-
-static LinkAnimationHeader* D_80853C5C[] = {
-    &gPlayerAnim_0030C8, &gPlayerAnim_0030C0, &gPlayerAnim_0030C0,
-    &gPlayerAnim_0030C8, &gPlayerAnim_0030C8, &gPlayerAnim_0030C8,
-};
-
-static LinkAnimationHeader* D_80853C74[] = {
-    &gPlayerAnim_0030D8, &gPlayerAnim_0030D0, &gPlayerAnim_0030D0,
-    &gPlayerAnim_0030D8, &gPlayerAnim_0030D8, &gPlayerAnim_0030D8,
-};
-
-static LinkAnimationHeader* D_80853C8C[] = {
-    &gPlayerAnim_0030B8, &gPlayerAnim_0030B0, &gPlayerAnim_0030B0,
-    &gPlayerAnim_0030B8, &gPlayerAnim_0030B8, &gPlayerAnim_0030B8,
-};
-
-static LinkAnimationHeader* D_80853CA4[] = {
-    &gPlayerAnim_002F20, &gPlayerAnim_002F18, &gPlayerAnim_002F18,
-    &gPlayerAnim_002F20, &gPlayerAnim_002F20, &gPlayerAnim_002F20,
-};
-
-static LinkAnimationHeader* D_80853CBC[] = {
-    &gPlayerAnim_002FF0, &gPlayerAnim_002FE8, &gPlayerAnim_002FE8,
-    &gPlayerAnim_002FF0, &gPlayerAnim_002FF0, &gPlayerAnim_002FF0,
-};
-
-static LinkAnimationHeader* D_80853CD4[] = {
-    &gPlayerAnim_003010, &gPlayerAnim_003008, &gPlayerAnim_003008,
-    &gPlayerAnim_003010, &gPlayerAnim_003010, &gPlayerAnim_003010,
-};
-
-static LinkAnimationHeader* D_80853CEC[] = {
-    &gPlayerAnim_003000, &gPlayerAnim_002FF8, &gPlayerAnim_002FF8,
-    &gPlayerAnim_003000, &gPlayerAnim_003000, &gPlayerAnim_003000,
-};
-
-static LinkAnimationHeader* D_80853D04[] = {
-    &gPlayerAnim_002EF0, &gPlayerAnim_002EE8, &gPlayerAnim_002EE8,
-    &gPlayerAnim_002EF8, &gPlayerAnim_002EF0, &gPlayerAnim_002EF0,
-};
-
-static LinkAnimationHeader* D_80853D1C[] = {
-    &gPlayerAnim_0031E0, &gPlayerAnim_0031D8, &gPlayerAnim_0031D8,
-    &gPlayerAnim_0031E8, &gPlayerAnim_0031E0, &gPlayerAnim_0031E0,
-};
-
-static LinkAnimationHeader* D_80853D34[] = {
-    &gPlayerAnim_003468, &gPlayerAnim_003438, &gPlayerAnim_003438,
-    &gPlayerAnim_003468, &gPlayerAnim_003468, &gPlayerAnim_003468,
+static LinkAnimationHeader* D_80853914[PLAYER_ANIMGROUP_MAX][PLAYER_ANIMTYPE_MAX] = {
+    /* PLAYER_ANIMGROUP_0 */
+    { &gPlayerAnim_003240, &gPlayerAnim_003238, &gPlayerAnim_003238, &gPlayerAnim_002BE0, &gPlayerAnim_003240,
+      &gPlayerAnim_003240 },
+    /* PLAYER_ANIMGROUP_1 */
+    { &gPlayerAnim_003290, &gPlayerAnim_003268, &gPlayerAnim_003268, &gPlayerAnim_002BF8, &gPlayerAnim_003290,
+      &gPlayerAnim_003290 },
```

Comment:
dont like what the formatter did here but its not in our control and will change when the anims are named. so w/e